proposed/2023/Add-dotnet-search.md

```diff
@@ -41,23 +44,91 @@ This option will specify a list of sources to search from. If a source is not sp
 #### **Option `-exact-match`**
 
 - This option will allow for users to be able to search and have only exact matches as an output.
+- The outputted list will contain all versions of the matching package ID
 - For example if a user uses `dotnet package search Newtonsoft.Json`
 
-Source: nuget.org
-| Package ID                                  | Latest Version | Authors | Downloads       |
-|---------------------------------------------|----------------|---------|-----------------|
-| Newtonsoft.Json                             | 13.0.3         |         | 3,829,822,911   |
-| Newtonsoft.Json.Bson                        | 1.0.2          |         | 554,641,545     |
-| Newtonsoft.Json.Schema                      | 3.0.15         |         | 39,648,430      |
-| Microsoft.AspNetCore.Mvc.NewtonsoftJson     | 7.0.12         |         | 317,067,823     |
-...
+        Source: nuget.org
+        | Package ID                                  | Latest Version | Authors | Downloads       |
+        |---------------------------------------------|----------------|---------|-----------------|
+        | Newtonsoft.Json                             | 13.0.3         |         | 3,829,822,911   |
+        | Newtonsoft.Json.Bson                        | 1.0.2          |         | 554,641,545     |
+        | Newtonsoft.Json.Schema                      | 3.0.15         |         | 39,648,430      |
+        | Microsoft.AspNetCore.Mvc.NewtonsoftJson     | 7.0.12         |         | 317,067,823     |
+        ...
 
 - Using ``dotnet package search Newtonsoft.Json --exact-match`` on the other side will have the following output
 
-Source: nuget.org
-| Package ID                                  | Latest Version | Authors | Downloads       |
-|---------------------------------------------|----------------|---------|-----------------|
-| Newtonsoft.Json                             | 13.0.3         |         | 3,829,822,911   |
+        Source: nuget.org
+        | Package ID                                  | Latest Version | Authors | Downloads       |
+        |---------------------------------------------|----------------|---------|-----------------|
+        | Newtonsoft.Json                             | 13.0.3         |         | 3,829,822,911   |
+
+#### **Option `--format`**
+
+This option will allow the specification of the output format. The option will take one of these three arguments: Table, List, or Json. The outputs will depend on verbosity. The following examples are for normal verbosity.
+
+- `--format Table` the output will be in a tabular form as shown below
```

Comment:
Do we want to have both list and table? 

I'm wondering because `dotnet list package` itself has 2 format options, json or console.

Reply:
fixed: `table` and `json` now